Solution Week-1/J1P2.py

```diff
@@ -1,8 +1,10 @@
 #2.Write a python script to enter any number and print the sum of its digit.
-def sum():
-    n=int(input("Enter a number : "))
-    s=0

```

Comment:
why you do this??

Reply:
Here, 
First of all I have created a user defined  function ”sum()” and then in the variable 'n' ... I have used two function namely int() and input() . 
int() function that type cast the inputted string into integer and input() function takes the input
Secondly I have used variable 's' and 
Intialized it with value zero as in this program I want to fund sum of it digits.. 

Hope you have found it helpful.. 
Thank you...